external/wycheproof/test.properties

```diff
@@ -0,0 +1,526 @@
+        script="thorntail-mp-tck.sh"
+        home_path="\${WORKDIR}"
+        test_results="testResults"
+        tag_version="2.6.0.Final"
+        debian_packages="git maven"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git maven"
+        centos_packages="git maven"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git maven"
+        ubi_minimal_packages="${ubi_packages}"
+        ;;
+    tomcat)
+        github_url="https://github.com/apache/tomcat.git"
+        script="tomcat-test.sh"
+        home_path=""
+        tag_version="10.0.0-M5"
+        ant_version="1.10.6"
+        openssl_version="1.1.1d"
+        debian_packages="git wget tar make gcc linux-libc-dev libc6-dev perl"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git wget tar make gcc libc-dev perl linux-headers"
+        centos_packages="git wget tar make gcc glibc-devel perl"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git wget tar make gcc glibc-devel"
+        ubi_minimal_packages="${ubi_packages}"
+        ;;
+    tomee)
+        github_url="https://github.com/apache/tomee.git"
+        script="tomee-test.sh"
+        home_path=""
+        test_results="testResults"
+        tag_version="tomee-8.0.2"
+        debian_packages="git maven"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git maven"
+        centos_packages="git maven"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git maven"
+        ubi_minimal_packages="${ubi_packages}"
+        ;;
+    wildfly)
+        github_url="https://github.com/wildfly/wildfly.git"
+        script="wildfly-test.sh"
+        home_path=""
+        test_results=""
+        tag_version="19.1.0.Final"
+        debian_packages="git maven"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git maven"
+        centos_packages="git maven"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git maven"
+        ubi_minimal_packages="${ubi_packages}"
+        ;;
+    wycheproof)
+        github_url="https://github.com/google/wycheproof.git"
+        script="wycheproof.sh"
+        home_path="\${WORKDIR}"
+        test_results="testResults"
+        tag_version="master"
+        bazel_version="1.2.1"
+        debian_packages="git wget unzip zip g++"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="bash git wget unzip zip g++ bash"
+        centos_packages="git wget unzip zip gcc-c++"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git wget unzip zip gcc-c++"
+        ubi_minimal_packages="${ubi_packages}"
```

Comment:
We only need this section for the test.properties.  Take a look at https://github.com/AdoptOpenJDK/openjdk-tests/pull/2452/files for a similar example.

Reply:
Thanks @smlambert for the review, working on the updates requested.